src/views/DefineCompany.vue

```diff
@@ -18,11 +18,52 @@
       </div>
     </section>
 
+    <section class="mt-10" v-show="isTypeCoop">
```

Comment:
Since this page is used for all entity types, but this section is coop-specific, how about creating a new sub-component for this? (Not now, don't want to drag out your ticket. Next time this code is touched.)

Also, what do you think about moving this text to the coop resource file? I'm on the fence for this, since this section is coop-specific anyway.

Reply:
For sure, i think encapsulating all of this makes sense. 

I thought about this, since this is coop specific we won't need any other variation of configuration for this afaik, so i'm not sure if it makes sense at this point. I think we should down the road, if design ends up asking for different verbiage or something based on different coop types maybe? Thats the only scenario that comes to mind.

Reply:
Two thumbs up.